src/util/system.h

```diff
@@ -195,6 +195,7 @@ class ArgsManager
     std::map<OptionsCategory, std::map<std::string, Arg>> m_available_args GUARDED_BY(cs_args);
     bool m_accept_any_command GUARDED_BY(cs_args){true};
     std::list<SectionInfo> m_config_sections GUARDED_BY(cs_args);
+    fs::path blocksPathCachedNetSpecific GUARDED_BY(cs_args);
```

Comment:
In commit "Change GetBlocksDir() to ArgsManager.GetBlocksDirPath()." (fa0b2291aa84fe372bb37a9fcbf1ad3165f7daea)

Again should follow naming convention, would suggest `m_cached_blocks_path`. See naming in  https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

Reply:
Applied, thanks.